packages/FormElement/src/components/Content/Content.js

```diff
@@ -26,9 +25,9 @@ function Content(props) {
   }
 
   return (
-    <sc.ContentContainer data-pka-anchor="form-element.content" {...moreProps}>
+    <div data-pka-anchor="form-element.content" {...moreProps}>
       {typeof children === "function" ? children({ idForLabel, refLabel, ariaDescribedBy }) : children}
-    </sc.ContentContainer>
+    </div>
```

Comment:
There was no need for the margin-bottom on the content so removed it. This is important if there is no description  or error components below the content then we don't get unwanted bottom margins still existing